ceph_deploy/hosts/__init__.py

```diff
@@ -61,6 +61,9 @@ def get(hostname,
     module.normalized_release = _normalized_release(release)
     module.distro = module.normalized_name
     module.is_el = module.normalized_name in ['redhat', 'centos', 'fedora', 'scientific']
+    module.is_rpm = module.normalized_name in ['redhat', 'centos',
+                                               'fedora', 'scientific', 'suse']
+    module.is_deb = not module.is_rpm
```

Comment:
This will be handy to have. It does  make me wonder what usecase we have where we are doing something module.is_el != module.is_rpm (meaning RPM, but not SUSE)...  Not important, just something to look at some day.  :)